addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1685,24 +1685,16 @@ options.registry.Carousel = options.Class.extend({
         // the slide overlay) + See "CarouselItem" option.
         this.$controls.addClass('o_we_no_overlay');
 
-        let _slideTimestamp;
-        this.$target.on('slide.bs.carousel.carousel_option', () => {
-            _slideTimestamp = window.performance.now();
-            setTimeout(() => this.trigger_up('hide_overlay'));
-        });
-        this.$target.on('slid.bs.carousel.carousel_option', () => {
-            // slid.bs.carousel is most of the time fired too soon by bootstrap
-            // since it emulates the transitionEnd with a setTimeout. We wait
-            // here an extra 20% of the time before retargeting edition, which
-            // should be enough...
-            const _slideDuration = (window.performance.now() - _slideTimestamp);
-            setTimeout(() => {
-                this.trigger_up('activate_snippet', {
-                    $snippet: this.$target.find('.carousel-item.active'),
-                    ifInactiveOptions: true,
-                });
-                this.$target.trigger('active_slide_targeted');
-            }, 0.2 * _slideDuration);
+        // Handle the sliding manually.
```

Comment:
(Commenting on a random line)

Carousel are broken on your branch:
![image](https://github.com/odoo/odoo/assets/10338094/82bbbd08-4346-447e-b364-3492767d2fac)

Reply:
I think it is because of the ZWS issue, it was happening on runbot too before the fixing PR was merged :wink: I will rebase :+1: